orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -47,7 +49,13 @@ class CreateWebhookTask implements RetryableTask {
     def customHeaders = stage.context.customHeaders
     boolean waitForCompletion = (stage.context.waitForCompletion as String)?.toBoolean()
 
-    def response = webhookService.exchange(method, url, payload, customHeaders)
+    def response
+    try {
+       response = webhookService.exchange(method, url, payload, customHeaders)
+    } catch (Exception e) {
```

Comment:
If it's a 429 (throttling) or 504 (gateway timeout), it seems like waiting 10 seconds and then retrying _would_ be the right thing to do.

We could restrict retries to a subset of status codes (429, 500 (generic error), 502 (bad gateway, maybe recoverable), 503 (service unavailable), 504), maybe?